### PR TITLE
Added image-to-text to Types.ts

### PR DESCRIPTION
**Pull request by merveenoyan**
cc: @osanseviero

### CODE REVIEW

**Comment by osanseviero** [comment]
One last thing to be added is an icon at https://github.com/huggingface/huggingface_hub/tree/main/widgets/src/lib/Icons and importing it in https://github.com/huggingface/huggingface_hub/blob/main/widgets/src/lib/ModelPipelineIcon/ModelPipelineIcon.svelte

You can find nice icons at https://icones.js.org/collection/carbon

Corresponding guide: https://huggingface.co/docs/hub/adding-a-task#ui-elements

**Comment by merveenoyan** [comment]
@osanseviero thanks!!! on it.

**Comment by merveenoyan** [comment]
Ekran Resmi 2021-12-08 18 53 14
it works on my local :p @osanseviero

**Review by osanseviero — APPROVED** [review]
Looks good!

File: widgets/src/lib/Icons/IconImageToText.svelte [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	/>
+    <path
+		d="M20 13v-2h-2.142a3.94 3.94 0 0 0-.425-1.019l1.517-1.517l-1.414-1.414l-1.517 1.517A3.944 3.944 0 0 0 15 8.142V6h-2v2.142a3.944 3.944 0 0 0-1.019.425L10.464 7.05L9.05 8.464l1.517 1.517A3.94 3.94 0 0 0 10.142 11H8v2h2.142a3.94 3.94 0 0 0 .425 1.019L9.05 15.536l1.414 1.414l1.517-1.517a3.944 3.944 0 0 0 1.019.425V18h2v-2.142a3.944 3.944 0 0 0 1.019-.425l1.517 1.517l1.414-1.414l-1.517-1.517A3.94 3.94 0 0 0 17.858 13zm-6 1a2 2 0 1 1 2-2a2.002 2.002 0 0 1-2 2z"
+		fill="currentColor"
+	/>
+    <path
+		d="M12 30H6a2.002 2.002 0 0 1-2-2V4a2.002 2.002 0 0 1 2-2h16a2.002 2.002 0 0 1 2 2v10h-2V4H6v24h6z"
+		fill="currentColor"
+	/>
+</svg>
```
Comment (mishig25) [inline widgets/src/lib/Icons/IconImageToText.svelte:28]: nit: could you add new line here

File: widgets/src/lib/ModelPipelineIcon/ModelPipelineIcon.svelte [hunk 2]
```diff
@@ -39,6 +40,7 @@
 		"table-question-answering": IconTableQuestionAnswering,
 		"question-answering": IconQuestionAnswering,
 		"zero-shot-classification": IconZeroShotClassification,
+		"image-to-text": IconImageToText,
```
Comment (mishig25) [inline widgets/src/lib/ModelPipelineIcon/ModelPipelineIcon.svelte:43]: nit: could you add this line after https://github.com/huggingface/huggingface_hub/blob/8bee737a94ee163eb9399e3af1247270c9c550a7/widgets/src/lib/ModelPipelineIcon/ModelPipelineIcon.svelte#L59

**Comment by merveenoyan** [comment]
@mishig25 nits are done

File: widgets/src/lib/Icons/IconImageToText.svelte [hunk 3]
```diff
@@ -24,5 +24,6 @@
     <path
 		d="M12 30H6a2.002 2.002 0 0 1-2-2V4a2.002 2.002 0 0 1 2-2h16a2.002 2.002 0 0 1 2 2v10h-2V4H6v24h6z"
 		fill="currentColor"
+	
 	/>
 </svg>
```
Comment (mishig25) [inline widgets/src/lib/Icons/IconImageToText.svelte]: oh my bad, I meant a new line at the end of the file (github highlighting made it look like I was asking new line at line 27)

otherwis, lgtm!

**Review by mishig25 — APPROVED** [review]
thanks for adding this!

**Comment by merveenoyan** [comment]
@mishig25 :')

**Comment by osanseviero** [comment]
There's a style error https://github.com/huggingface/huggingface_hub/runs/4506443553?check_suite_focus=true

**Comment by osanseviero** [comment]
Given the netlify issue is not related to your PR, I don't think you should close it :) instead we should figure what's going on with the authorization

**Comment by merveenoyan** [comment]
@osanseviero it's a fork and my account doesn't have the credentials that's why imo, no?

**Comment by osanseviero** [comment]
@mishig25 is it possible to deploy from netlify when having a PR from a fork? I'm a bit surprised it doesn't work

**Pull request closed without merging**